From: =?utf-8?q?IOhannes_m_zm=C3=B6lnig?= <umlaeute@debian.org>
Date: Tue, 29 Jan 2019 11:31:36 +0100
Subject: search for GUI plugins in /etc/pd/plugins-enabled

Origin: Debian
Last-Update: 2016-02-17

this gives us a standard path where the user can symlink-enable their
system-wide GUI plugins.
Last-Update: 2016-02-17
---
 tcl/pd-gui.tcl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcl/pd-gui.tcl b/tcl/pd-gui.tcl
index a611b71..e0418af 100755
--- a/tcl/pd-gui.tcl
+++ b/tcl/pd-gui.tcl
@@ -791,7 +791,7 @@ proc load_startup_plugins {} {
     load_plugin_script [file join $::sys_guidir pd_docsdir.tcl]
 
     # load other installed plugins
-    foreach pathdir [concat $::sys_temppath $::sys_searchpath $::sys_staticpath] {
+    foreach pathdir [concat $::sys_temppath $::sys_searchpath "/etc/pd/plugins-enabled" $::sys_staticpath] {
         set dir [file normalize $pathdir]
         if { ! [file isdirectory $dir]} {continue}
         if { [ catch {
